api/views.py

```
@@ -0,0 +1,1021 @@
+                denial_id=denial_id
+            )
+        if denial_opt is None:
+            raise Exception("Denial not found")
+        denial: Denial = denial_opt  # type: ignore
+        appeal = None
+        try:
+            appeal = Appeal.filter_to_allowed_appeals(current_user).get(
+                for_denial=denial, pending=True
+            )
+        except Appeal.DoesNotExist:
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Appeal.DoesNotExist)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Appeal.DoesNotExist)`

(SIM105)